docs/installation.rst

```diff
@@ -66,7 +66,6 @@ Start by making the following changes to your ``settings.py`` file.
    # Add 'mozilla_django_oidc' authentication backend
    AUTHENTICATION_BACKENDS = (
        # ...
```

Comment:
I'd remove those `...` at the start and bottom and I'd add some text explaining that it's suggested that you only keep `ModelBackend`